hackathon_site/dashboard/frontend/src/pages/Acknowledgement/Acknowledgement.tsx

```diff
@@ -0,0 +1,130 @@
+    const {
+        error: getAcceptanceError,
+        isLoading,
+        user: acceptanceUser,
+    } = useSelector(userAcceptanceSelector);
+    const {
+        error: createProfileError,
+        isLoading: createProfileLoading,
+        profile,
+    } = useSelector(createProfileSelector);
+    const userDoesNotHaveRole = userType === "none";
+    const [showAcknowledgements, setShowAcknowledgements] = useState(false);
+
+    useEffect(() => {
+        const today = new Date();
+        if (hardwareSignOutStartDate > today) {
+            push("/404");
+        } else if (userDoesNotHaveRole) {
+            dispatch(fetchUserAcceptanceStatus());
+        }
+    }, [dispatch, userDoesNotHaveRole]);
+
+    return (
+        <>
+            <Header showNavbar={false} />
+            <Container maxWidth="lg">
+                {getAcceptanceError ? (
+                    <AlertBox
+                        error={getAcceptanceError?.message ?? getAcceptanceError}
+                        title="An error has occurred"
+                        type="error"
+                    />
+                ) : userDoesNotHaveRole ? (
+                    isLoading ? (
+                        <LinearProgress data-testid="userReviewStatusLoadingBar" />
+                    ) : !showAcknowledgements ? (
+                        <UserAcceptanceMessage
+                            status={acceptanceUser?.review_status ?? "Incomplete"}
+                            handleGetStarted={() => setShowAcknowledgements(true)}
+                        />
+                    ) : createProfileError ? (
+                        <AlertBox
+                            error={createProfileError?.message ?? createProfileError}
+                            title="An error has occurred while granting you permissions"
+                            type="error"
+                        />
+                    ) : createProfileLoading ? (
+                        <CircularProgress />
+                    ) : profile ? (
+                        <Fade in={!!profile} timeout={1000}>
+                            <SuccessMessage />
+                        </Fade>
+                    ) : (
+                        <Fade in={showAcknowledgements} timeout={1000}>
+                            <div>
+                                <AcknowledgementForm isLoading={false} />
+                            </div>
+                        </Fade>
+                    )
+                ) : (
+                    <UserHasBeenGrantedAccessMessage />
+                )}
+            </Container>
+        </>
+    );
+};
+
+const SuccessMessage = () => {
+    const user = useSelector(userSelector);
+    const { profile } = useSelector(createProfileSelector);
+
+    return (
+        <Grid container justifyContent="center">
+            <Grid item lg={6} xs={12}>
+                <Card style={{ padding: "15px 5px" }}>
+                    <Typography style={{ fontSize: "25px" }} align="center">
```

Comment:
Styling can be improved. I can make the Figma design if necessary.

Minor thing: we can capitalize the first letter of user's name so that it looks professional. Sometimes when user fills their application, they don't pay attention to the capital letter. (i.e I applied with my name as leo li, and on the website, it should display Leo there to make it look nice).

Reply:
updated the naming